src/main.js

```diff
@@ -154,27 +156,28 @@ async function getAllActiveItemsOnTheBoard() {
       }
     });
   } catch (error) {
-    core.setFailed(`An error occurred retrieving the cards: ${error}`);
+    core.setFailed(`An error occurred retrieving the cards: ${error.message}`);
   }
   return [];
 }
 
 async function getActiveBranches() {
-  try {
-    const response = await octokit.rest.repos.listBranches({
+  await octokit
```

Comment:
So I could be being super dumb right now, it is the morning after all, but I think we need this return statement added right? The old return statements got shoved into the anonymous function being passed to the `then` method.
```suggestion
  return await octokit
```

Reply:
Yeah we need a fix for that.  Good catch!

Reply:
This is the exact thing I was worried I would miss when I converted them all from try/catch to await/then/catch.

Reply:
I made the change in a slightly different way but it's been pushed up.

Reply:
Your change matches the pattern of the other actions, so I think it makes more sense!